llama-index-integrations/vector_stores/llama-index-vector-stores-qdrant/tests/test_vector_stores_qdrant.py

```diff
@@ -58,3 +60,33 @@ async def test_aclear(vector_store: QdrantVectorStore) -> None:
                 "33333333-3333-3333-3333-333333333333",
             ]
         )
+
+
+def test_parse_query_result(vector_store: QdrantVectorStore) -> None:
```

Comment:
Oh, I kind of meant calling `get_nodes()` and ensuring that it works (But this is good too!)